CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [3.0.1] - 2023-09-13
+
+### Added
+
+- Funding files and updates in the Readme about funding the project.
+
+### Changed
+
+- The `WordPress.Security.EscapeOutput` sniff has more modular error codes, so that developers can ignore certain output escaping cases based on where the errors occur. Props [@anomiex].
+- Updated the CI workflow to test the example ruleset for issues.
+
+### Fixed
+
+- Fixed the sniff name in the `phpcs.xml.dist.sample` file (case-sensitive sniff name). Props [@dawidurbanski].
```

Comment:
```suggestion
- Fixed a sniff name in the `phpcs.xml.dist.sample` file (case-sensitive sniff name). Props [@dawidurbanski].
```

Whether this belongs in "Fixed" or "Changed" is open for debate. It's not a bug in a sniff (end-user functionality), but was a "bug" in the documentation, which _if strictly copied_ could affect end-users, so I'm not sure...

Reply:
It is a change, but also a fix from a bug that kinda affected the users 🤷🏼‍♂️ Not a sniff bug, but issue was reported.